com.abisource.AbiWord.json

```diff
@@ -0,0 +1,218 @@
+{
+    "app-id" : "com.abisource.AbiWord",
+    "branch" : "3.0",
+    "runtime" : "org.gnome.Sdk",
+    "runtime-version" : "3.28",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "abiword",
+    "tags" : [
+       "stable"
+    ],
+    "finish-args" : [
+        "--allow=devel",
```

Comment:
what do you need this for?

Reply:
I think this is leftover from Gnome Builder. I copied the .json over. Removed for the next push.